pa

```diff
@@ -199,6 +195,14 @@ usage() {
 }
 
 main() {
+    command="$1"
+
+    # Combine the rest of positional arguments into
+    # a name and remove control characters from it
+    # so that a name can always be safely displayed.
+    shift
+    name=$(printf %s "$*" | LC_ALL=C tr -d '[:cntrl:]')
```

Comment:
additional important change that's not strictly related to the issue, but makes use of a single $name assignment added in this PR, is stripping out control characters from password name, so that things like `pa add te\nst` don't mess with `pa list`